util/src/main/java/io/kubernetes/client/informer/cache/ProcessorListener.java

```diff
@@ -64,6 +64,9 @@ public void run() {
       } catch (InterruptedException e) {
         log.error("processor interrupted: {}", e.getMessage());
         return;
+      } catch (Throwable t) {
```

Comment:
Hrm, I don't love this here, I assume that this is because we're running a custom user handler and it could throw anything. Perhaps we can just wrap the handler call in the catch of Throwable?

Reply:
exactly. i wrap the throwable-catching block only for handler invocation